### PR TITLE
samples: peripheral_hids_mouse: Fix Cmake warning with NRF_RPC

### PR DESCRIPTION
**Pull request by dchat-nordic**
Fix the NRF_RPC_THREAD_STACK_SIZE value assigned when NRF_RPC is not used.

Jira: NCSDK-20740

### CODE REVIEW

File: samples/bluetooth/peripheral_hids_mouse/Kconfig [hunk 1]
```diff
@@ -29,3 +29,10 @@ config BT_DIRECTED_ADVERTISING
 	depends on BT_HIDS_SECURITY_ENABLED
 
 endmenu
+
+if NRF_RPC
+
+config NRF_RPC_THREAD_STACK_SIZE
```
Comment (KAGA164) [inline samples/bluetooth/peripheral_hids_mouse/Kconfig:35]: This does not work for me. Thread stack size is still 1024 bytes

**Review by KAGA164 — CHANGES_REQUESTED** [review]

File: samples/bluetooth/peripheral_hids_mouse/overlay-nrf_rpc.conf [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#
```
Comment (KAGA164) [inline samples/bluetooth/peripheral_hids_mouse/overlay-nrf_rpc.conf:1]: What do you think about naming it just nrf_rpc.conf?

File: samples/bluetooth/peripheral_hids_mouse/README.rst [hunk 3]
```diff
@@ -68,6 +68,9 @@ The HID service specification does not require encryption (:kconfig:option:`CONF
 
 Building and running
 ********************
+
+To build this sample with :ref:`lib_nrf_rpc` on `nrf5340dk`, set ``OVERLAY_CONFIG`` to :file:`overlay-nrf_rpc.conf`.
```
Comment (KAGA164) [inline samples/bluetooth/peripheral_hids_mouse/README.rst:72]: Could you put console code block with example command call for it under this line?

File: samples/bluetooth/peripheral_hids_mouse/overlay-nrf_rpc.conf (hunk 2 above)
Comment (dchat-nordic) [reply samples/bluetooth/peripheral_hids_mouse/overlay-nrf_rpc.conf:1]: I think overlay should stay. It is consistent with other samples. Naming it nrf_rpc.conf would imply that it's a separate config, so all the Kconfig options would need to be copied there.

Comment (KAGA164) [reply samples/bluetooth/peripheral_hids_mouse/overlay-nrf_rpc.conf:1]: Ok then

**Review by KAGA164 — APPROVED** [review]

File: samples/bluetooth/peripheral_hids_mouse/README.rst [hunk 4]
```diff
@@ -68,6 +68,13 @@ The HID service specification does not require encryption (:kconfig:option:`CONF
...
```
Comment (annakielar) [inline samples/bluetooth/peripheral_hids_mouse/README.rst:72]:
```suggestion
To build this sample with the :ref:`nrf_rpc_ipc` library on the nRF5340 DK, set :makevar:`OVERLAY_CONFIG` option to the :file:`overlay-nrf_rpc.conf` file.
```

**Review by annakielar — CHANGES_REQUESTED** [review]

**Review by maje-emb — APPROVED** [review]

File: samples/bluetooth/peripheral_hids_mouse/README.rst [hunk 5]
```diff
@@ -68,6 +68,13 @@ The HID service specification does not require encryption (:kconfig:option:`CONF
 
 Building and running
 ********************
+
+To build this sample with the :ref:`nrf_rpc_ipc` library on the nRF5340 DK, set :makevar:`OVERLAY_CONFIG` option to the :file:`overlay-nrf_rpc.conf` file.
```
Comment (annakielar) [inline samples/bluetooth/peripheral_hids_mouse/README.rst:72]:
```suggestion
To build this sample with the :ref:`nrf_rpc_ipc_readme` library on the nRF5340 DK, set :makevar:`OVERLAY_CONFIG` option to the :file:`overlay-nrf_rpc.conf` file.
```

**Review by annakielar — APPROVED** [review]

File: doc/nrf/releases/release-notes-changelog.rst [hunk 6]
```diff
@@ -216,6 +216,7 @@ Bluetooth samples
 * :ref:`peripheral_hids_mouse` sample:
 
   * The :kconfig:option:`CONFIG_BT_SMP` Kconfig option is included when ``CONFIG_BT_HIDS_SECURITY_ENABLED`` is selected.
+  * Fixed the CMake warning by moving the nRF RPC configuration (the :kconfig:option:`CONFIG_NRF_RPC_THREAD_STACK_SIZE` Kconfig option) to a separate overlay config file.
```
Comment (alwa-nordic) [inline doc/nrf/releases/release-notes-changelog.rst:219]:
```suggestion
  * Fixed a CMake warning by moving the nRF RPC configuration (the :kconfig:option:`CONFIG_NRF_RPC_THREAD_STACK_SIZE` Kconfig option) to a separate overlay config file.
```

**Review by alwa-nordic — APPROVED** [review]

**Pull request merged**